x-pack/plugins/lists/server/services/exception_lists/utils.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObject } from 'kibana/server';
+
+import {
+  ExceptionListItemSchema,
+  ExceptionListSchema,
+  ExceptionListSoSchema,
+} from '../../../common/schemas';
+import {
+  SavedObjectType,
+  exceptionListAgnosticSavedObjectType,
+  exceptionListSavedObjectType,
+} from '../../saved_objects';
+
+import { NamespaceType } from './types';
+
+export const getSavedObjectType = ({
+  namespaceType,
+}: {
+  namespaceType: NamespaceType;
+}): SavedObjectType => {
+  if (namespaceType === 'agnostic') {
+    return exceptionListAgnosticSavedObjectType;
+  } else {
+    return exceptionListSavedObjectType;
+  }
+};
+
+export const transformSavedObjetToExceptionList = ({
```

Comment:
Small typo here "SavedObjet" ... missing the 'c' in 'object'.

Reply:
Thanks did a search and replace everywhere for it and pushed it up.